pkg/cmd/workspace/create.go

```diff
@@ -96,17 +97,20 @@ var CreateCmd = &cobra.Command{
 			log.Fatal("workspace name and repository urls are required")
 			return
 		}
-
-		visited := make(map[string]bool)
+		visited := make(map[string]int)
 
 		for i := range projects {
 			if projects[i].Source == nil || projects[i].Source.Repository == nil || projects[i].Source.Repository.Url == nil {
 				log.Fatal("Error: repository url is required")
 			}
-			if visited[*projects[i].Source.Repository.Url] {
-				log.Fatalf("Error: duplicate repository url: %s", *projects[i].Source.Repository.Url)
+
+			visited[*projects[i].Source.Repository.Url]++
```

Comment:
This name correction should be moved to the TUI.

When you select 2 of the same repos and then enter the advanced config view (using F10), there's no way of differentiating those 2 projects.

Reply:
fixed!